src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -156,6 +158,12 @@ NewTerminalArgs Pane::GetTerminalArgsForPane() const
     // object. That would work for schemes set by the Terminal, but not ones set
     // by VT, but that seems good enough.
 
+    // Only fill in the ContentGuid if absolutely needed.

```

Comment:
why? is it expensive?

Reply:
(isn't it just a number?)

Reply:
If you fill in a number here (even 0), we'll serialize that number, AND treat that action as an "attach existing" rather than a "create"